src/classes/EMPReplicantProfile.cls

```diff
@@ -0,0 +1,104 @@
+                                                                    From EMP_Farm_Follow_Up__c
+                                                                    where Farm_diagnosis_and_targets__r.Farm__r.EMP_Farm__c =:organizationMemberId.Id];
+    		if (farmFollowUps.size() > 0) {
+	         	return farmFollowUps;
+	        }
+	    }
+	    return null;
+	}
+
+	public List<EMP_Technical_Assistance__c> getTechnicalAssistance() {
+		if (organizationMember != null) {
+        	EMP_Technical_Assistance__c[] farmTechnicalAssistance = [Select Date__c, EMP_Technical_Assistance__c, EMP_Technical_Assistance_Consultant__c, Identification_Number__c, NIT__c, Organization_Name__c, Productive_Sector__c, Recommendations_Next_Visit__c, Situation_Property__c, Visit_Number__c, Visit_To__c, Which__c
+                                                                    From EMP_Technical_Assistance__c
+                                                                    where EMP_Technical_Assistance__r.EMP_Farm__c =:organizationMemberId.Id];
+    		if (farmTechnicalAssistance.size() > 0) {
+	       		return farmTechnicalAssistance;
+	   		}
+		}
+		return null;
+	}
+
+	public List<EMP_Farm_Baseline__c> getBaseline() {
+		if (organizationMember != null) {
+        	List<EMP_Farm_Baseline__c> farmBaseline = [Select Agrochemicals_Disposal__c, Amount__c, BPA_Certified__c, Children_Participation__c, Commercialization_Agreements__c, Commercialization_Location__c, Crop_Product__c, Crop_Rotation__c, Cycle_Applications__c, Economic_Activity__c, EMP_Farm_Baseline__c, Employees__c, Employees_Training__c, Growing_Infrastructure__c, Income_Composition__c, Institution_Type__c, Irrigation_System__c, Irrigation_System_Origin__c, Minimal_Tillage__c, Mobile_Phone__c, Monthly_Income__c, Num_Employees__c, Organization_Name__c, Packages__c, Permanent_Employees__c, Population_Type__c, Price_Kilo__c, Processing_Product__c, Processing_Product_Commercialization__c, Processing_Product_Location__c, Processing_Type__c, Production_Costs__c, Productive_Sector__c, Products_Traded__c, Reference_Total_Area__c, Requires_Credit__c, Scholarship_Level__c, Slope_Planting__c, Sowing_Area__c, Temporary_Employees__c, Total_Cultivated_Area__c, Triple_Washed__c, Under_Eighteen__c, User_Credit__c, Vulnerable_population__c, Wastewater_Management__c, Water_Concession__c
+                                                                    From EMP_Farm_Baseline__c
+                                                                    where EMP_Farm_Baseline__r.EMP_Farm__c =:organizationMemberId.Id ];
+    		if (farmBaseline.size() > 0) {
+	       		return farmBaseline;
+	   		}
+		}
```

Comment:
I don't know why it looks different between Sublime & Git, but I'm going to change it and include each column on a separate line (in all methods). Thanks